src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -71,22 +71,18 @@ export default class ReserveSubmitButton extends Vue {
     this.cancelAnalyzeName('NamesCapture')
   }
 
-  getData (type: string): any {
-    if (this.getAssumedName) type = 'assumed'
-    let data: any
+  getData (type: NrType): DraftReqI | ConditionalReqI | ReservedReqI {
+    if (this.getAssumedName) type = NrType.ASSUMED
     switch (type) {
-      case 'assumed':
-      case 'draft':
-        data = this.getDraftNameReservation
-        break
-      case 'conditional':
-        data = this.getConditionalNameReservation
-        break
-      case 'reserved':
-        data = this.getReservedNameReservation
-        break
+      case NrType.ASSUMED:
```

Comment:
This may* be obsolete. I know it is an option to edit an NR to `assumed` via name examination ui but i'm not sure Name Request supports it anymore. I could be wrong tho!

Reply:
No action required, just thinking!

Reply:
Yes, I think some of this is obsolete. In this method, only CONDITIONAL and RESERVED are ever called. In the similar method (getNrTypeData) in actions.ts, only DRAFT is ever called.
This could be cleaned up later, unless you really want me to clean it up now :)

Reply:
No no, no biggie! 
Just encountered the same thing previously when looking at the Edits.